src/Apps/Artist/Components/ArtistCollectionsRail/__tests__/ArtistCollectionsRail.test.tsx

```diff
@@ -136,6 +136,7 @@ describe("CollectionsRail", () => {
           subject: "clicked next button",
           type: "Button",
         })
+        done()
```

Comment:
Is there a reason this test uses the `done()` mechanism versus marking this test `async` and doing `await flushPromiseQueue()` before the assertion, which the rest do?

I wouldn't hold up a merge for this, but if it is an arbitrary choice, I'd prefer consistency with the other tests to increase the likelihood of that pattern being followed.

Reply:
Eh, mostly just because I left it after I noticed how many were there. There are *alot* of tests that use setTimeout (with varying amounts of time). I think the async/await pattern is a lot more legible but it'd be a fair amount more work that warrants it's own PR.